app/controllers/decision_reviews_controller.rb

```diff
@@ -291,6 +304,11 @@ def create_change_history_csv(filter_params)
     end
   end
 
+  def create_change_history_individual_report(task_id)
+    events = ClaimHistoryService.new(business_line, task_id: task_id).build_events
+    events
```

Comment:
Does this even really need to be a method if it's just a one liner? Unless it was because code climate was yelling about some nonsense. You could also wrap the metrics call into this method and pull it out of the controller action to clean it up a bit.

```suggestion
    ClaimHistoryService.new(business_line, task_id: task_id).build_events
```